blockchain_api/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 relay_rpc = { path = "../relay_rpc" }
-reqwest = "0.11"
+reqwest = { version = "0.12.2", features = ["json"] }
```

Comment:
Can we use the version without `patch` number? That will allow us to update to the latest patch version.

Reply:
See convo [here](https://github.com/WalletConnect/utils-rs/pull/16#discussion_r1556610453) it does allow updating the latest version.